### PR TITLE
Feature/box model

### PR DESCRIPTION
**Pull request by augusticor**
The box model exercise, i set the width of the nav by padding feature

### CODE REVIEW

**Comment by augusticor** [comment]
Hello, this branch has some extra files since I made the pull command from the original project, so the changes you did in some files weren't into this forked repo, and when i push the changes it send them with these files. 🤔 That's why.

File: 2-positioning/styles.css [hunk 1]
```diff
@@ -3,27 +3,27 @@ body {
     padding: 0;
 }
 
-header {
-    position: sticky;
-    border: 1px solid green;
+.footer {
```
Comment (pushdev-code) [inline 2-positioning/styles.css:6]: why are these changes here? did you include positioning tweaks to the exercise?

File: 2-positioning/styles.css [hunk 2]
```diff
@@ -3,27 +3,27 @@ body {
...
+    border: 1px solid green;
     color: white;
+    position: sticky;
 }
 
-nav {
-    position: absolute;
+.navig {
     border: 1px solid blue;
+    position: absolute;
```
Comment (pushdev-code) [inline 2-positioning/styles.css:21]: if it's absolute positioning, at least two of these properties need to be specified: top, right, bottom, left.

File: 2-positioning/styles.css [hunk 3]
```diff
@@ -3,27 +3,27 @@ body {
...
     width: 200px;
```
Comment (pushdev-code) [inline 2-positioning/styles.css:22]: I don't see any other element set to relative, therefore this nav will be positioned at the top of the window, overlapping the header. double check.

File: 3-box-model/styles.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+.header {
+    background-color: lightskyblue;
+    border: 1px solid green;
+    color: white;
+    position: sticky;
+}
+
+.list {
+    width: 100%;
```
Comment (pushdev-code) [inline 3-box-model/styles.css:26]: sort these properties alphabetically.

File: 3-box-model/styles.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    margin: 0;
```
Comment (pushdev-code) [inline 3-box-model/styles.css:27]: has this element any margin?

File: 3-box-model/styles.css [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.navig {
+    border: 1px solid blue;
+    margin: 0;
+    position: absolute;
```
Comment (pushdev-code) [inline 3-box-model/styles.css:33]: it seems this element is overlapping the header, isn't it?

File: 2-positioning/styles.css (hunk 1 above)
Comment (augusticor) [reply 2-positioning/styles.css:6]: I included something that was missing

File: 3-box-model/styles.css (hunk 6 above)
Comment (augusticor) [reply 3-box-model/styles.css:33]: No, it isn't overlapping the header
